<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>572</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Longin</string>
  </property>
  <property name="iconSize">
   <size>
    <width>30</width>
    <height>30</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>-30</x>
      <y>300</y>
      <width>601</width>
      <height>201</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget{
background :rgb(233, 243, 255);
}</string>
    </property>
    <widget class="QLabel" name="generator_label">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>50</y>
       <width>171</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Password Generator :</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="generator_lineEdit">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>50</y>
       <width>281</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
background :rgb(255, 255, 255);
border : 2px solid rgb(230, 230, 230);
border-radius :20px;
}</string>
     </property>
    </widget>
    <widget class="QPushButton" name="copy_pushButton">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>110</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background : rgb(255, 255, 255);
border-radius : 10px;
}</string>
     </property>
     <property name="text">
      <string>Copy</string>
     </property>
    </widget>
    <widget class="QPushButton" name="redo_pushButton">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>60</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../icon/redo.png</normaloff>../icon/redo.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>571</width>
      <height>301</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget{
background : rgb(241, 241, 241);
}</string>
    </property>
    <widget class="QPushButton" name="login_pushButton">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>210</y>
       <width>151</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
color: rgb(44, 157, 255);

}</string>
     </property>
     <property name="text">
      <string>Login</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="password_label">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>150</y>
       <width>101</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Password :</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="password_lineEdit">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>160</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
background : rgb(255, 255, 255);
border-radius : 10px;
}</string>
     </property>
    </widget>
    <widget class="QLabel" name="username_label">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>70</y>
       <width>101</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Username :</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="username_lineEdit">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>80</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
background : rgb(255, 255, 255);
border-radius : 10px;
}</string>
     </property>
    </widget>
    <zorder>password_lineEdit</zorder>
    <zorder>login_pushButton</zorder>
    <zorder>password_label</zorder>
    <zorder>username_label</zorder>
    <zorder>username_lineEdit</zorder>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
